jenkins/run-unittests

```diff
@@ -1,10 +1,3 @@
 #!/bin/bash
 
-# For the CI jobs, we run unit tests expecting most of the environment to be
-# provided by the container image, and only a few modules not installable via
-# RPM need to be added (see jenkins/python-setup.sh).  We then instruct tox to
-# use the current environment (not setting up a virtualenv), and to run the
-# tests using whatever version of the python interpreter the install provides
-# (the `py` directs Tox to use whatever it finds).
```

Comment:
While I grant that the old comment is no longer needed, I think we still need a comment here:  there is a reason that we wrap this command in a script, and we need to point that out, so that no one like me is tempted to try to move this command into the pipeline file and get rid of this "silly little script".

Reply:
No need for this script anymore.

Reply:
Because it has been superseded and replaced by `jenkins/runner`?...